airflow-core/src/airflow/ui/src/i18n/locales/de/assets.json

```diff
@@ -0,0 +1,29 @@
+{
+  "consumingDags": "Konsumierende Dags",
+  "createEvent": {
+    "button": "Ereignis erstellen",
+    "manual": {
+      "description": "Direkt ein Ereignis zu Datensets (Asset) erstellen",
+      "extra": "Extra Daten zu Ereignis",
+      "label": "Manuell"
+    },
+    "materialize": {
+      "description": "Das Dag das diesen Datensatz (Asset) produziert direkt auslösen",
```

Comment:
Datenset oder Datensatz? Beides wird mehrmals verwendet in diesem PR.

I think standardising on "Datenset" makes more sense, but not a strong opinion other than I think it should be the same term everywhere :)

Reply:
Uuups, hier bin ich vom Begriff einmal falsch abgebogen und dann habe ich es konsistent "falsch" gemacht. Danke! Habe es wieder konsistent zu "Datenset" gemacht.